### PR TITLE
Parameterize probe values for Jaeger-chart

### PR DESCRIPTION
**Pull request by olovholm**
Issue #, if available: 390

Description of changes:
* Parameterize liveness and readiness probes
* Sets default values in Chart to Kubernetes default values

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: stable/appmesh-jaeger/values.yaml [hunk 1]
```diff
@@ -20,6 +20,23 @@ resources:
...
+    periodSeconds: 10
+    timeoutSeconds: 1
+    successThreshold: 1
+    failureThreshold: 3
+  readiness:
+    initialDelaySeconds: 0
+    periodSeconds: 10
+    timeoutSeconds: 1
+    successThreshold: 1
+    failureThreshold: 3
```
Comment (fawadkhaliq) [inline stable/appmesh-jaeger/values.yaml:38]: Can you please also update the README configuration section [1] with new the params

[1] https://github.com/aws/eks-charts/blob/master/stable/appmesh-jaeger/README.md#configuration

File: stable/appmesh-jaeger/values.yaml [hunk 2]
```diff
@@ -20,6 +20,23 @@ resources:
     cpu: 100m
     memory: 256Mi
 
+  ## Prometheus server readiness and liveness probe initial delay and timeout
```
Comment (fawadkhaliq) [inline stable/appmesh-jaeger/values.yaml:23]: typo: Prometheus?

**Review by fawadkhaliq — CHANGES_REQUESTED** [review]
@olovholm thanks for the pull request. Looks good overall. Added a couple of comments

**Review by fawadkhaliq — APPROVED** [review]
lgtm

**Comment by olovholm** [comment]
@fawadkhaliq : Thanks for review. Branch now merged, but could we have a release? From the build jobs it doesn't look like the helm charts are deployed, and I cannot find the new version in the chart-repository.

**Pull request merged**